public/app/plugins/panel/table2/module.tsx

```diff
@@ -1,25 +1,25 @@
 import { PanelPlugin } from '@grafana/data';
 import { TablePanel } from './TablePanel';
-import { CustomFieldConfig, defaults, Options } from './types';
+import { CustomFieldConfig, Options } from './types';
 
 export const plugin = new PanelPlugin<Options, CustomFieldConfig>(TablePanel)
-  .setDefaults(defaults)
   .useFieldConfig({
     useCustomConfig: builder => {
       builder
         .addNumberInput({
           path: 'width',
-          name: 'Column width',
+          name: 'Width',
           description: 'column width (for table)',
           settings: {
             placeholder: 'auto',
             min: 20,
             max: 300,
           },
+          category: ['Column'],
```

Comment:
not sure this category makes sense, all field options will be column options :)  and there are only two.